engine/src/test/java/com/redhat/cloud/notifications/processors/drawer/DrawerProcessorTest.java

```diff
@@ -100,6 +102,9 @@ void shouldNotProcessWhenEndpointsAreEmpty() {
 
     @Test
     void shouldCreateTwoDrawerNotifications() {
+        // Make sure that the payload does not get stored in the database.
+        Mockito.when(this.engineConfig.getKafkaToCamelMaximumRequestSize()).thenReturn(Integer.MAX_VALUE);
```

Comment:
Since `kafkaToCamelMaximumResquestSize` value is back to `10485760` by default, we shouldn't need this.
(Same for `EventingTypeProcessorTest` and `WebhookTest`)

Reply:
It seems that we need to keep it for the `EventingTypeProcessorTest` and the `WebhookTest`, since otherwise it forces us to create the event in the database along with all the parents: event type, application and bundle. I guess we can keep it like this for those two classes, what do you think?